README.md

```diff
@@ -24,7 +24,7 @@ Cattage supports the following versions:
 
 | Cattage | Argo CD | Accurate |
 |---------|---------|----------|
-| 0.1.0   | 2.2.x   | 0.3.x    | 
+| 0.1.x   | 2.2.5   | 0.3.x    | 
```

Comment:
```suggestion
| 0.1.x   | 2.3.2   | 0.3.x    | 
```

Reply:
@yokaze 
I intend to support argocd v2.3.x in cattage v0.2.